specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/capacityReservationExamples/CapacityReservation_Update_MaximumSet_Gen.json

```diff
@@ -19,7 +19,7 @@
         }
       },
       "sku": {
-        "name": "DSv3-Type1",
+        "name": "Standard_DS1_v2",
         "tier": "aaa",
         "capacity": 7
```

Comment:
we don't use the tier property in request or response, so we should remove it from the example.

Reply:
"tier": "aaa" is still there in the example, pls. remove as we don't use it.